controllers/openstackdataplanenode_controller.go

```diff
@@ -363,6 +365,7 @@ func (r *OpenStackDataPlaneNodeReconciler) GenerateInventory(ctx context.Context
 		}
 		cm.Data = map[string]string{
 			"inventory": string(invData),
+			"network":   string(networkConfig.Template),
```

Comment:
Yeah, the only reason I chose to append to this one instead of creating a new one is due to the design of https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/pkg/deployment/deployment.go#L79

I didn't want to add a new param to each and every call to `ConditionalDeploy`. 

We can rename it, although since we're just defining a template file used by Ansible, I thought it was still essentially in-line with everything else that configmap already contained. 

If the preference is to rename though, I can do that.